src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -50,7 +79,7 @@ public void removeListener(@NotNull Listener listener) {
   }
 
   private static boolean isGlobalFlutterSdkSet() {
-    return FlutterSdk.getGlobalFlutterSdk() != null;
+    return FlutterSdk.getGlobalFlutterSdk() != null && FlutterSdkUtil.isFluttering();
```

Comment:
Also, at a quick glance it looks like `isFluttering` is often (if not always) checked before calling this method?  So maybe it's not necessary?  (But then you put it here for some reason! ;) )